src/components/Loader/LoadingText.css

```diff
@@ -0,0 +1,57 @@
+.loading-text.loading-text__small,
```

Comment:
Use the .dui- prefix for main style class names. Eg: `.dui-loading-text` https://github.com/decentraland/ui/blob/d4582593dcd053d3bc654b7195f5f451ac368f94/src/components/NFTCard/NFTCard.css#L1C2-L1C5